.gitignore

```diff
@@ -338,4 +338,5 @@ ASALocalRun/
 .localhistory/
 
 # BeatPulse healthcheck temp database
-healthchecksdb
\ No newline at end of file
+healthchecksdb
+/CliFx.Tests.Dummy/Properties/launchSettings.json
```

Comment:
Do we need this?

Reply:
Looks like `healthchecksdb` got added by VS automatically. Doesn't look like we use it, so I'll remove it so it doesn't clutter the file.
I'll also remove the `launchSettings.json` file and `.gitignore` entry.

Reply:
I was referring just to `/CliFx.Tests.Dummy/Properties/launchSettings.json`